graph/types/schema.graphqls

```diff
@@ -92,6 +93,20 @@ type Mutation {
 
   addDiscussionTags(discussionID: ID!, tags:[String!]): [Tag!]
   deleteDiscussionTags(discussionID: ID!, tags:[String!]): [Tag!]
+
+  # Add flair accessibility to private discussions
+  addDiscussionFlairTemplatesAccess(discussionID: ID!, flairTemplateIDs:[ID!]): [DiscussionFlairTemplateAccess!]
```

Comment:
Let's return discussion here and expose the flair template access objects on the discussion.